eus_cddlib/README.md

```diff
@@ -0,0 +1,3 @@
+# eus_cddlib
+
+Use cddlib (https://inf.ethz.ch/personal/fukudak/cdd_home/) from EusLisp for finding vertices of convex polytopes.
```

Comment:
please explain example program (eus_cddlib/euslisp/test-eus-cddlib.l) , what is input and what is output, `test-eus-cddlib.l` seems to show output as images, so please add that image to README.md